onyo/lib/onyo.py

```diff
@@ -124,6 +125,10 @@ def set_config(self,
         if self.version == '1' and key == 'onyo.assets.name-format':
             key = 'onyo.assets.filename'
 
+        # clear the config cache
+        self._config_cache: dict[Path, dict[str, str]] = dict()
```

Comment:
Type annotation is incorrect here. (`Path`)

Reply:
Oh nice catch. I fixed the other one, but forgot this. Thanks. :-)